src/HatchIntakeSubsystem.cpp

```diff
@@ -1,40 +1,54 @@
 #include <HatchIntakeSubsystem.h>
+#include <CORERobotlib.h>
 
 #include "Robot.h"
 
 HatchIntakeSubsystem::HatchIntakeSubsystem() : m_hatchIntake(HATCH_INTAKE_MOTOR_PORT),
-                                               m_hatchIntakeSolenoid(HATCH_INTAKE_SOLENOID_IN_PORT, HATCH_INTAKE_SOLENOID_OUT_PORT),
-                                               m_photoeye(HATCH_INTAKE_PHOTOEYE)
+                                               m_hatchActuator(HATCH_INTAKE_SOLENOID_PORT),
+                                               m_photoeye(HATCH_INTAKE_PHOTOEYE),
+                                               m_actuatorBottomLimit("Bottom limit of the hatch intake"),
+                                               m_actuatorTopLimit("Top limit of the hatch intake")
 {
-    isDown = false;
 }
 
 void HatchIntakeSubsystem::robotInit(){
     operatorJoystick->RegisterButton(CORE::COREJoystick::JoystickButton::RIGHT_BUTTON);
     operatorJoystick->RegisterButton(CORE::COREJoystick::JoystickButton::RIGHT_TRIGGER);
+    
+    m_hatchIntake.SetNeutralMode(NeutralMode::Coast);
+    m_hatchActuator.SetNeutralMode(NeutralMode::Brake);
 
+    m_hatchActuator.ConfigSelectedFeedbackSensor(FeedbackDevice::CTRE_MagEncoder_Relative, 0, 0);
+    m_hatchActuator.SetSelectedSensorPosition(0, 0, 0);
+    
+    //Uncertain of the necessity of the line below
+    m_hatchActuator.SetSensorPhase(true);
 }
 
 void HatchIntakeSubsystem::teleopInit(){
    
 }
 
 void HatchIntakeSubsystem::teleop(){
-    if(operatorJoystick->GetButton(CORE::COREJoystick::JoystickButton::RIGHT_BUTTON)){
+    if (operatorJoystick->GetButton(CORE::COREJoystick::JoystickButton::RIGHT_BUTTON)){
         SetIntakeOn();
-    }else{
+    } else {
         SetIntakeOff();
```

Comment:
Maybe combine the set intake off and on functions so that you can pass in a bool that tells you whether or not you want to set the intake on?